core/src/accounts_background_service.rs

```diff
@@ -30,8 +34,8 @@ impl AccountsBackgroundService {
 
                 bank.process_dead_slots();
 
-                // Currently, given INTERVAL_MS, we process 1 slot/100 ms
-                bank.process_stale_slot();
+                consumed_budget = bank
+                    .process_stale_slot_with_budget(consumed_budget, SHRUNKEN_ACCOUNT_PER_INTERVAL);
```

Comment:
Can we remove Bank::process_stale_slot now?

Reply:
Yeah! Removed